validator/src/main/java/io/avaje/validation/core/DValidator.java

```diff
@@ -32,6 +32,7 @@ final class DValidator implements Validator, ValidationContext {
   private final MessageInterpolator interpolator;
   private final LocaleResolver localeResolver;
   private final DTemplateLookup templateLookup;
+  private final Map<String, String> messageCache = new HashMap<>();
```

Comment:
Does this need to be a ConcurrentHashMap ?

Reply:
sure ok